platforms/Linux/DEB/Ubuntu/shared/debian/swiftlang.postinst.in

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+update-alternatives --install /usr/bin/swift swiftlang /usr/lib/swiftlang/X.Y.Z/bin/swift-frontend 50 \
```

Comment:
In the absence of multiple versions of the package that provide an alternative, the automatically installed alternative here gets installed as `/usr/bin/swift` etc. Once multiple Swift package versions exist, the one exposed as `/usr/bin/swift` etc is configurable with `update-alternatives --config swiftlang`.

If this is an acceptable solution, it probably should be documented in the root level README in the repo at least?